modules/ocf_lb/manifests/keepalived.pp

```diff
@@ -0,0 +1,19 @@
+class ocf_lb::keepalived(
+  Array[String] $virtual_addresses,
+  Array[String] $virtual_addresses_v6,
+  String $secret_lookup,
+  Integer[1, 255] $vrid
+) {
+  $secret = lookup($secret_lookup)
+
+  package { 'keepalived':; } ->
+  # IPv6 addresses have to be specified separately as they cannot be in the vrrp
+  # packet together (keepalived 1.2.20+) so they need to be in a
+  # virtual_ipaddress_excluded block instead.
```

Comment:
I'd say this comment makes more sense in the actual template itself rather than here